sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusTargetScaler.cs

```diff
@@ -94,7 +94,17 @@ internal TargetScalerResult GetScaleResultInternal(TargetScalerContext context,
                 throw new ArgumentOutOfRangeException($"Unexpected concurrency='{concurrency}' - the value must be > 0.");
             }
 
-            int targetWorkerCount = (int)Math.Ceiling(messageCount / (decimal)concurrency);
+            int targetWorkerCount;
+
+            try
```

Comment:
Why does the Service Bus implementation use a different approach than the Event Hubs implementation for the same scenario?